Tests/Framework/Graphics/IndexBufferTest.cs

```diff
@@ -90,5 +90,13 @@ public void NullDeviceShouldThrowArgumentNullException()
             });
             GC.GetTotalMemory(true); // collect uninitialized IndexBuffer
         }
+
+        [Test]
+        [RunOnUI]
+        public void TypedConstructorShouldWork()
+        {
+            var indexBuffer = new IndexBuffer(gd, typeof(short), 12, BufferUsage.None);
```

Comment:
I would have added a test for `typeof(int)` as well 🤷

Reply:
Good call.